checker/jdk/nullness/src/java/util/ArrayList.java

```diff
@@ -290,7 +290,7 @@ public boolean isEmpty() {
      * @return <tt>true</tt> if this list contains the specified element
      */
     @Pure
-    public boolean contains(@Nullable Object o) {
```

Comment:
This should stay `@Nullable` because `null` is allowed according to the Javadoc. That's true for all the changes in this file.